### ISSUE TITLE
Plans for Openssl 3.0 ?

### ISSUE THREAD

**Issue by Chaolyu14** [cisco/libacvp#644]
Hello

are there any plans of supporting Openssl 3.0 ?

Best Regards
Richard

**Comment by bfussell** [comment]
Absolutely. Last time I checked openssl.org indicated that 3.0.0 was supposed to be API compatible with 1.1.1 so there shouldn't be a ton of work. We have not tested with 3.0.0 so I'm not sure what issues exist currently.

Note that openssl.org will be FIPS testing using their own ACVP implementation much like they did back in 1.0.2 using their CAVS test harness. Are there certain OEs you are looking to test that they are not testing ?

**Comment by iamwum** [comment]
Hi,
Any update on openssl 3.0.0 support ?
